ginga/examples/configs/channel_Image.cfg

```diff
@@ -0,0 +1,118 @@
+#
+# Image channel preferences file
+#
+# Place this in file under ~/.ginga with the name "channel_Image.cfg" for
+# channel name "Image". This file WILL BE OVERWRITTEN by the Preferences
+# plugin (see below). NOT ALL PARAMETERS IN GUI CAN BE ADJUSTED HERE.
```

Comment:
It is a bit confusing to me that this config file does not address everything in `Preferences` and vice versa. So I added notes here in the comments. Please correct me if I misunderstood.